errors.lua

```diff
@@ -382,6 +382,84 @@ local function netbox_call(conn, func_name, ...)
     )
 end
 
+
+local function _wrap_future_result(prefix, suffix, ...)
+    local args = {...}
+    local res, err = args[1], args[2]
+    if res ~= nil then
+        return res
+    end
```

Comment:
Seems you don't need varargs here. Just use `_wrap_future_result(prefix, suffix, res, err)`

Reply:
Thanks, fixed